scripts/build.sh

```diff
@@ -12,12 +12,20 @@ bundle='algoliasearch.helper'
 
 echo "Build"
 
-browserify index.js -s algoliasearchHelper -o dist/algoliasearch.helper.js
+browserify index.js \
+  -s algoliasearchHelper \
+  --debug | \
+  exorcist dist/algoliasearch.helper.js.map > dist/algoliasearch.helper.js
```

Comment:
exorcist doesn't accept arguments about input and output, so it's forced to be like this